src/pages/editor/curators/OrganizationCuratorRow.tsx

```diff
@@ -0,0 +1,107 @@
+import AddLinkIcon from '@mui/icons-material/AddLink';
+import AdjustIcon from '@mui/icons-material/Adjust';
+import EditIcon from '@mui/icons-material/Edit';
+import EventIcon from '@mui/icons-material/Event';
+import MarkEmailReadIcon from '@mui/icons-material/MarkEmailRead';
+import SchoolIcon from '@mui/icons-material/School';
+import TaskIcon from '@mui/icons-material/Task';
+import { Box, IconButton, Typography } from '@mui/material';
+import { ComponentType, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { InstitutionUser, RoleName } from '../../../types/user.types';
+import { UserFormDialog } from '../../basic_data/institution_admin/edit_user/UserFormDialog';
+
+interface OrganizationCuratorRowProps {
+  curator: InstitutionUser;
+  refetchCurators: () => void;
+}
+
+const curatorRolesConfig = [
+  { rolename: RoleName.SupportCurator, color: 'generalSupportCase.main', SelectedIcon: MarkEmailReadIcon },
+  { rolename: RoleName.PublishingCurator, color: 'publishingRequest.main', SelectedIcon: TaskIcon },
+  { rolename: RoleName.CuratorThesis, color: 'publishingRequest.main', SelectedIcon: SchoolIcon },
+  { rolename: RoleName.CuratorThesisEmbargo, color: 'publishingRequest.main', SelectedIcon: EventIcon },
+  { rolename: RoleName.DoiCurator, color: 'doiRequest.main', SelectedIcon: AddLinkIcon },
+  { rolename: RoleName.NviCurator, color: 'nvi.main', SelectedIcon: AdjustIcon },
+] satisfies {
+  rolename:
+    | RoleName.SupportCurator
+    | RoleName.PublishingCurator
+    | RoleName.CuratorThesis
+    | RoleName.CuratorThesisEmbargo
+    | RoleName.DoiCurator
+    | RoleName.NviCurator;
```

Comment:
Tar et lite triks med `satisfies` her, for å sikre at `rolename` får riktig type, og vi slipper å måtte brke `t<any>(`editor.curators.role.${rolename}`)` på linje 91